src/components/sideBar/index.js

```diff
@@ -0,0 +1,21 @@
+import useTranslation from 'hooks/useTranslation';
+import React, { Component } from 'react';
```

Comment:
Are you using this elements that you've just imported here? If not, WDYT if you remove them?

Reply:
done